### PR TITLE
LPS-89474 Adds missing override

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by darquesdev** [comment]
Hey @p2kmgcl we're going to split this PR in 2 (back and front). DO NOT REVIEW IT!

@ealonso @andresfulla

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js [hunk 1]
```diff
@@ -646,5 +647,27 @@ const INITIAL_STATE = {
 
 };
 
-export {INITIAL_STATE};
+/**
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/state.es.js]: Same thing here. If this default state values bring any concern, we can just leave it out.

**Comment by p2kmgcl** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/67652. [See changes here.](https://github.com/p2kmgcl/liferay-portal/compare/aab29c598e5b~5...aab29c598e5b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by p2kmgcl** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2143. [See changes here.](https://github.com/p2kmgcl/liferay-portal/compare/aab29c598e5b~5...aab29c598e5b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**